test/e2e/persistent_volumes.go

```diff
@@ -114,14 +114,36 @@ func deletePVCandValidatePV(c *client.Client, ns string, pvc *api.PersistentVolu
 	return pv, pvc, nil
 }
 
+// Wraps deletePVCandValidatePV() to delete and validate 1:1 PV-PVC pairs
+// Takes the list of pvs and list of pvcs
+// Returns updated pvList, pvcList, and error
+func deleteMultiplePVCandValidatePV(c *client.Client, ns string, pvcList []*api.PersistentVolumeClaim, pvList []*api.PersistentVolume) ([]*api.PersistentVolume, []*api.PersistentVolumeClaim, error) {
+
+	var err error
+
+	for pvi, pv := range pvList {
+		for pvci, pvc := range pvcList {
+			if pvcList[pvci] != nil && len(pvc.Name) > 0 {
```

Comment:
1) if a nil test is necessary, see (2), why index the slice? Why not just test `if pvc != ni`l ?
2) Does the check for nil indicate that the pvcList can be sparse? And if so why isn't there a similar check for the pvList?